src/main/java/pl/edu/agh/iet/katabank/Account.java

```diff
@@ -60,4 +60,12 @@ private void checkValidAmount(BigDecimal amount) {
                     + (amount == null ? null : amount.toString()));
         }
     }
+
+    public void transfer(Account targetAccount, BigDecimal bigDecimal) {
+        if (targetAccount == null) {
+            throw new RuntimeException("Target account cannot be null.");
+        }
+        this.withdraw(bigDecimal);
+        targetAccount.deposit(bigDecimal);
+    }
 }
```

Comment:
Hi. We have the method tested with cucumber - with correct inputs. Please unit test with targetAccount being null.